support/generate_bindings.cr

```diff
@@ -224,8 +224,8 @@ platforms.each_with_index do |platform, idx|
     args: args,
     env: env,
     shell: false,
-    output: true,
-    error: true,
+    output: STDOUT,
```

Comment:
This would be better as `Process::Redirect::Inherit` to keep the exact old behaviour.